src/config.py

```diff
@@ -1,8 +1,28 @@
 import os
 import yaml
-from typing import Dict, Any
+from typing import Dict, Any, List, Optional
```

Comment:
```suggestion
from typing import Dict, Any, Optional
```

Are Dict and Any used?

Reply:
Not anymore, after changing the annotation for `load_config`. Removed!